### DISCUSSION TITLE
Setting ulimit to a elasticsearch container

### DISCUSSION THREAD

**Discussion by marciomanske** [Trendyol/jest-testcontainers#34] [Q&A]
Hello!

Is there any way to set extra parameters for the container setup ? For instance, the ulimits like in the docker-compose file ?

...
    ulimits:
     memlock:
        soft: -1
        hard: -1
...

For the java version of testcontainers, we have the method "withCreateContainerCmdModifier" that is applied before the container creation.
I have a test where I need to start three containers: postgres, rabbitmq, and elasticsearch. As rabbitmq can't be started using the docker-compose file, I have to use the jest-testcontainers-config.js to describe the containers, and then, I couldn't find a way to set the ulimits.

Thanks!

**Comment by Yengas** [comment]
Hey @marciomanske,

Currently the supported configurations for the non docker-compose files can be seen [here](https://github.com/Trendyol/jest-testcontainers/blob/474677b4a9ca16603ea96362f0b837d2c5b41341/src/containers.ts#L78). We don't support ulimits right now. This could be a nice addition to the library. Please feel free to create an MR if you have the time and the team can review / merge it.
